arch/arm/src/sama5/sam_tc.c

```diff
@@ -1458,4 +1461,4 @@ int sam_tc_divisor(uint32_t frequency, uint32_t *div, uint32_t *tcclks)
   return OK;
 }
 
-#endif /* CONFIG_SAMA5_TC0 || CONFIG_SAMA5_TC1 || CONFIG_SAMA5_TC2 */
+#endif /* CONFIG_SAMA5_TC0 || CONFIG_SAMA5_TC1 || CONFIG_SAMA5_TC2 */
```

Comment:
revert the change?

Reply:
Not sure what I changed or you want reverted?

Reply:
Oh - CR missing at the end. No idea how that happened...although it built Ok and passed checkpatch.sh?